subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1688,6 +1688,50 @@ static void le_read_tx_power(struct net_buf *buf, struct net_buf **evt)
 	ll_tx_pwr_get(&rp->min_tx_power, &rp->max_tx_power);
 }
 
+#if IS_ENABLED(CONFIG_BT_CTLR_DF)
+#if IS_ENABLED(CONFIG_BT_CTLR_DF_CONN_CTE_RSP)
+static void le_df_set_conn_cte_tx_params(struct net_buf *buf,
+					 struct net_buf **evt)
+{
+	struct bt_hci_cp_le_set_conn_cte_tx_params *cmd = (void *)buf->data;
+	struct bt_hci_rp_le_set_conn_cte_tx_params *rp;
+	uint16_t handle = sys_le16_to_cpu(cmd->handle);
+	uint8_t status;
+
```

Comment:
```suggestion
	uint16_t handle, handle_le16;
	uint8_t status;

	handle_le16 = cmd->handle;
	handle = sys_le16_to_cpu(handle_le16);
```

Reply:
Wouldn't it be better to use just `cmd->handle` instead of allocation of new variable and assign it to rp->handle directly?

Reply:
cmd is reused for event generation, the value may have been overwritten.

Reply:
Ok, code changed due @cvinayak explanations provided outside of github. Here is PR that does similar fix in other places: https://github.com/zephyrproject-rtos/zephyr/pull/30248/files

I'd like to leave a comment here, that, in my humble opinion, the issue should be solved in another way in long term. Use of unwritten policy: "do not use `cmd->XXX` after call to `hci_cmd_complete` should be temporary solution.